### PR TITLE
fix(registry): fix openid identifier match (case-insensitive)

### PR DESCRIPTION
**Pull request by stas-nc**

### CODE REVIEW

File: registry/server/auth.ts [hunk 1]
```diff
@@ -318,10 +318,14 @@ export default async (app: Express, settingsService: SettingsService, config: an
...
-        identifier,
-    });
+    const user = await db
+        .select()
+        .from('auth_entities')
+        .first('identifier', 'id', 'role', 'secret')
+        .where({
+            provider,
+        })
+        .andWhereRaw('LOWER(identifier) = LOWER(?)', [identifier]);
```
Comment (b1ff) [inline registry/server/auth.ts:328]: won't it break mysql?
Also is it effective?

**Review by yehor-manzhula — APPROVED** [review]

File: registry/server/auth.ts (hunk 1 above)
Comment (stas-nc) [reply registry/server/auth.ts:328]: Tests on sqlite and mysql pass
Additional test for case-insensitive identifier added.

**Review by b1ff — APPROVED** [review]

File: registry/server/auth.ts (hunk 1 above)
Comment (stas-nc) [reply registry/server/auth.ts:328]: Regarding effectiveness https://stackoverflow.com/a/7005332/5825564
Can add an index on lower(identity column) though I'm not sure that this query and table is used very much.

Comment (b1ff) [reply registry/server/auth.ts:328]: Ok, lets see how it will affect DB load

File: e2e/spec/news.spec.e2e.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ Scenario('should open a news page and show news sources', async ({ I, newsPage:
     I.see('Pick a news source', newsPage.bannerHeadline);
 });
 
-Scenario.skip('should open an article page from a direct link', async ({ I, newsPage: newsPage }) => {
+Scenario.only('should open an article page from a direct link', async ({ I, newsPage: newsPage }) => {
```
Comment (yehor-manzhula) [inline e2e/spec/news.spec.e2e.ts:15]: is only expected here?

Comment (stas-nc) [reply e2e/spec/news.spec.e2e.ts:15]: sorry, not ready yet

**Review by yehor-manzhula — APPROVED** [review]

**Pull request merged**